usecases/user/login.md

```diff
@@ -38,16 +38,32 @@
         <td>
             <ul>
                 <li>
-                    Item 1
+                    The <a href="../user.md"> user </a> enters the login page
                 </li>
                 <li>
-                    Item 2
+                    The <a href="../user.md"> user </a> provides his login credentials and the server details
                 </li>
                 <li>
-                    Item 3
+                    The system verifies the login credentials
                 </li>
                 <li>
-                    Item 4
+                    if the credentials are valid:
```

Comment:
move exceptions to exceptions section;
Scenario --> always "everything goes right case"